examples/CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ project(hipsycl-examples)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../cmake)
 
-find_package(hipSYCL CONFIG REQUIRED)
+find_package(hipSYCL CONFIG)
+
+if(NOT hipSYCL_FOUND)
+  message(WARNING "Not found hipSYCL cmake integration retrying with default path: /opt/hipSYCL/lib/cmake/")
+  set(hipSYCL_DIR /opt/hipSYCL/lib/cmake/)
+  find_package(hipSYCL CONFIG REQUIRED)
+endif()
+set(hipSYCL_DIR /opt/hipSYCL/lib/cmake/ CACHE PATH
```

Comment:
It may be more intuitive for the user to just have that point to `/opt/hipSYCL` and then add `lib/cmake` automatically. After all, the subdirectory where the cmake files are stored will be the same for every hipSYCL installation.